README.md

```diff
@@ -12,15 +12,16 @@ Setup an ALB with related resources.
 
 ### Available variables
 
-* [`name_prefix`]: String(required): Name prefix of the ALB and security group
+* [`name_prefix`]: String(optional): Name prefix of the ALB and security group
+* [`name`]: String(optional): Name prefix of the ALB and security group in the format var.project-var.environment-var.name
```

Comment:
Technically speaking this is a "name suffix" or "name component". "name prefix" might be a bit misleading. Also for the ELB module.